### DISCUSSION TITLE
Development concerns

### DISCUSSION THREAD

**Discussion by louiss0** [ElMassimo/iles#148] [General]
I'm not here to put too much pressure on you at all. But I need to understand a few things before making a decision on whether to use this one or Astro. The reason why I ask these questions is that I don't like being an early adopter of fresh technology. 

1. When is 1.0.0 going to be released?
2. What are the future plans for this tool will it just be an SSG tool?
3. Are you going to build a team around this framework or are you going to work on it occasionally?

**Comment by ElMassimo** [comment]
> I don't like being an early adopter of fresh technology.

In that case, I'd suggest you use Nuxt instead.

Please see https://github.com/ElMassimo/iles/discussions/12#discussioncomment-3011751 and https://github.com/ElMassimo/iles/discussions/55#discussioncomment-1756480.

**Reply by louiss0** [reply]
You seem to have a long roadmap to 1.0.0, I'd suggest you make your own dev tools for ^iles instead of relying on the Vue ones. That way framework mixing can happen without too much hassle.
